BranchMonsterFactory/ExternalLibs/Branch-SDK/BNCConfig.h

```diff
@@ -1,34 +0,0 @@
-//
```

Comment:
Also if this is now Cocoapods, should we include the .xcworkspace file or is that just going to confuse users?